pages/osx/spotify.md

```diff
@@ -0,0 +1,28 @@
+# shpotify
```

Comment:
```suggestion
# spotify
```
Typo.

Reply:
This was intentional, shpotify is the actual name of the project. Now that I think of it, this might be confusing so it would probably be better to change it to the name of the executable.

Reply:
Yes, we usually have the name of the executable there.